test/Renderer/ImageTest.php

```diff
@@ -38,7 +38,8 @@ public function testType()
     public function testGoodImageResource()
     {
         $imageResource = imagecreatetruecolor(1, 1);
-        $this->renderer->setResource($imageResource);
+        $renderer = $this->renderer->setResource($imageResource);
+        $this->assertSame($this->renderer, $renderer);
```

Comment:
In general I am fine with these changes, but probably better, and more future-proof, will be to add annotation [`@doesNotPerformAssertions`](https://phpunit.readthedocs.io/en/7.1/annotations.html#doesnotperformassertions) instead of testing fluent interface.

We tend to drop fluent interfaces and it will be done also here for the next major release.

Reply:
Okay, changed that.